amplitude/client.go

```diff
@@ -1,61 +1,108 @@
 package amplitude
 
+import (
+	"github.com/amplitude/analytics-go/amplitude/internal"
```

Comment:
Not sure why we are importing from github link, should we import with relative local path here?

Reply:
Go doesn't support relative paths